package-builder/extensions/apm/build.sh

```diff
@@ -6,7 +6,7 @@ echo "Building apm for gcp-php${SHORT_VERSION}"
 
 PNAME="gcp-php${SHORT_VERSION}-apm"
 
-if [ ${SHORT_VERSION} == '56' ]; then
+if [ ${SHORT_VERSION} == '56' ] || [ ${SHORT_VERSION} == '80' ]; then
     echo "apm extension only for PHP 7.0+, relies on removed json.h for PHP 5.6"
```

Comment:
Seems like this message should be updated to say it doesn't work on `8.0`, as well. It only references `5.6` but with the additional condition it's no longer accurate.

Reply:
I could change the message to say `apm extension only for PHP 7.*, relies on removed json.h for PHP 5.6`